airflow/providers/google/cloud/hooks/datacatalog.py

```diff
@@ -18,16 +18,18 @@
 from typing import Dict, Optional, Sequence, Tuple, Union
 
 from google.api_core.retry import Retry
-from google.cloud.datacatalog_v1beta1 import DataCatalogClient
-from google.cloud.datacatalog_v1beta1.types import (
+from google.cloud import datacatalog
+from google.cloud.datacatalog_v1beta1 import (
+    CreateTagRequest,
+    DataCatalogClient,
     Entry,
     EntryGroup,
-    FieldMask,
     SearchCatalogRequest,
     Tag,
     TagTemplate,
     TagTemplateField,
 )
+from google.protobuf import field_mask_pb2 as field_mask  # type: ignore
```

Comment:
```suggestion
from google.protobuf.field_mask_pb2 import FieldMask
```

Should we keep it unified with import in other places?

Reply:
It just copied these lines from the Google library, but it will delete the alias, because it is not needed.